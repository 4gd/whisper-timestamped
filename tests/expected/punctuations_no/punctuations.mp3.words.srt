1
00:00:00,380 --> 00:00:01,300
Dis-moi

2
00:00:01,300 --> 00:00:01,460
est-ce

3
00:00:01,460 --> 00:00:01,640
que

4
00:00:01,640 --> 00:00:02,000
l'avion

5
00:00:02,000 --> 00:00:02,400
vole

