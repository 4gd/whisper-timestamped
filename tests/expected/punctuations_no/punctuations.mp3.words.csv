Dis-moi,0.38,1.3
est-ce,1.3,1.46
que,1.46,1.64
l'avion,1.64,2.0
vole,2.0,2.4
